### PR TITLE
About update

### PR DESCRIPTION
**Pull request by karczujr**
Section done

### CODE REVIEW

File: styles/sass/main.scss [hunk 1]
```diff
@@ -13,6 +13,23 @@
   --black: #000;
 }
 
+.grid {
```
Comment (ArturBa) [inline styles/sass/main.scss]: this might be called `grid-2-col`

File: styles/sass/main.scss [hunk 2]
```diff
@@ -13,6 +13,23 @@
...
+  grid-gap: 48px;
+
+  @include breakpoint(tablet) {
+    grid-template-columns: 1fr;
+    width: auto;
+    height: auto;
+  }
+}
+
+.small-body-decoration {
```
Comment (ArturBa) [inline styles/sass/main.scss:28]: you might use `@extend .small-body;` here. 
Then there will be no need to wrap a `p` inside a `div` to get the same result 

(you might need to add `padding-inline-start: 24px` in that case)
It's up to you ;)

File: src/components/About.astro [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        Porsche i&nbspwielu innych.
+      </p>
+    </div>
+
+    <div class="small-body-decoration">
+      <p class="small-body">Na naprawę udzielamy 12 miesięcy gwarancji.</p>
+    </div>
+  </div>
+  <div>
+    <img class="img-about-container" src="img/about.jpg" alt="" />
```
Comment (ArturBa) [inline src/components/About.astro:23]: all images might be replaced with Astro's [Image](https://docs.astro.build/en/guides/assets/)  for better performance. 

It's experimental for now, but might be worth a try (in another task)

**Review by ArturBa — APPROVED** [review]

File: styles/sass/main.scss (hunk 2 above)
Comment (karczujr) [reply styles/sass/main.scss:28]: Done

**Pull request merged**